ntfy/cli.py

```diff
@@ -49,9 +49,10 @@ def send_notification(message, args, config):
         module = import_module('ntfy.backends.{}'.format(backend))
 
         try:
+            backend_config = config.get(backend, {})
+            backend_config.update(args.option)
```

Comment:
Something like this perhaps:
backend_config.update(dict([(k,literal_eval(v)) for k,v in args.option.items()]))